R/post_process_plot.R

```diff
@@ -45,6 +45,13 @@ post_process_plot <- function(plot, is_sorted = TRUE, horizontal = TRUE,
 
   }
 
+  if (other & !is.null(threshold)) {
+    caption <-  paste(
+      "'Other' contains all categories with a value less than or equal to",
+      threshold, ".")
```

Comment:
Could you use `paste0` instead? This will add a space before the final `.` which does not look good. In addition, please put the trailing `)` on its own line.